README.md

```diff
@@ -165,3 +165,4 @@ module.exports = {
 | sep | `String` | `'/'` | Separator for `namespaces` and `module`. |
 | localeInterpolate | `RegExp` or `RegExp[]` or `String` or `String[]` | `[/_[a-z]{2}_[A-Z]{2}\./, /_[a-z]{2}\./, /[/\\][a-z]{2}_[A-Z]{2}[/\\]/, /[/\\][a-z]{2}[/\\]/]` | Uses to detect `locale` in `resourcePath`. |
 | defaultLocale | `String` | `'en_US'` | Uses `defaultLocale` if `localeInterpolate` fails to detect it. |
+| requireAngular | `Boolean` | `false` | indicates whether to add `var angular = require("angular")` to generated code or not. defaults to 'False'. |
```

Comment:
introducing `requireAngular` option (which actually doesn't work yet :D)

Reply:
@Ardeshir81 I would suggest to set it to `true` to keep it backward compatible.